os/include/tinyara/lcd/st7701sn.h

```diff
@@ -40,6 +40,9 @@
 #define MIPI_LCD_LIMIT		500
 #define MIPI_FRAME_RATE         60
 #define MIPI_LANE_NUMBER        2
+#define LCD_HEIGHT_MM           93.6
+#define LCD_WIDTH_MM            56.16
+#define LCD_SIZE_INCH           4.3

```

Comment:
these lcd information and mipi-dsi setting value are common form

so we need to make structure about this and feel the values in each lcd code.

```c
struct mipi_dsi_lcd {
    uint_8 hsa;
    uint_8 hbp;
...
}

struct lcd_device {
    uint_8 height_mm;
    uint_8 width_mm;

```

Reply:
okay so how about below?

```c
struct lcd_data st7785 = {
    XPixels = CONFIG_LCD_XRES,
    YPixels = CONFIG_LCD_YRES,
    mipi_frame_rate = 60,
    mipi_dsi_RTNI = 4.
    mipi_dsi_HSA = 30,
...
};
```
We need to create the same macro.
It is better to fill the struct because it already has struct than this method.